src/handlers/v1/markets/[chainId]/[marketId]/zap-supply/index.ts

```diff
@@ -163,7 +163,9 @@ export const v1GetZapSupplyQuotationRoute: Route<GetZapSupplyQuotationRouteParam
         );
       }
       const targetBorrowUSD = new BigNumberJS(borrowUSD);
-      const targetLiquidationThreshold = common.formatBigUnit(targetLiquidationLimit.div(targetCollateralUSD), 4);
+      const targetLiquidationThreshold = !targetLiquidationLimit.isZero()
```

Comment:
Is it better to replace it with `!targetCollateralUSD.isZero()`?

Reply:
I believe using `targetLiquidationLimit` is better because
- the calculation is based on `targetLiquidationLimit` which is the root cause of `NaN` (dividend is zero)
- `targetLiquidationLimit` is calculated by `liquidationLimit` derived from `collateralUSD`. This implies that `targetLiquidationLimit` equals zero when `targetCollateralUSD` is zero.
- check `targetCollateralUSD` here would be a little bit confusing if reviewers do not know the relation between `targetCollateralUSD` and  `targetLiquidationThreshold`

I understand that the intention is to skip the calculation when no collateral exists, but I believe using `targetLiquidationLimit` is more straightforward.

Reply:
I think that directly checking whether the divisor is zero is the better solution. The root cause is that dividing by zero leads to the NaN issue, and by avoiding division by zero, we can prevent the problem altogether. This approach is simpler and more straightforward, without the need to consider extensive context and background knowledge.

@zodahu, what are your thoughts on this?